lib/matplotlib/animation.py

```diff
@@ -1586,47 +1571,44 @@ def gen_function() -> obj
-       first frame.
+        A function used to draw a clear frame. If not given, the results of
+        drawing from the first item in the frames sequence will be used. This
+        function will be called once before the first frame.
 
-       The required signature is::
+        The required signature is::
 
-          def init_func() -> iterable_of_artists
+            def init_func() -> iterable_of_artists
 
-       If ``blit == True``, *init_func* must return an iterable of artists
-       to be re-drawn. This information is used by the blitting
-       algorithm to determine which parts of the figure have to be updated.
-       The return value is unused if ``blit == False`` and may be omitted in
-       that case.
+        If ``blit == True``, *init_func* must return an iterable of artists
+        to be re-drawn. This information is used by the blitting algorithm to
+        determine which parts of the figure have to be updated.  The return
+        value is unused if ``blit == False`` and may be omitted in that case.
 
     fargs : tuple or None, optional
-       Additional arguments to pass to each call to *func*.
+        Additional arguments to pass to each call to *func*.
 
     save_count : int, optional
-       The number of values from *frames* to cache.
+        The number of values from *frames* to cache.
 
-    interval : number, optional
-       Delay between frames in milliseconds.  Defaults to 200.
+    interval : int, default: 200
+        Delay between frames in milliseconds.
 
-    repeat_delay : number, optional
-       If the animation in repeated, adds a delay in milliseconds
-       before repeating the animation.  Defaults to *None*.
+    repeat_delay : int, default: 0
+        The delay in milliseconds between consecutive animation runs, if
+        *repeat* is True.
 
-    repeat : bool, optional
-       Controls whether the animation should repeat when the sequence
-       of frames is completed.  Defaults to *True*.
+    repeat : bool, default: True
+        Whether the animation repeats when the sequence of frames is completed.
 
-    blit : bool, optional
-       Controls whether blitting is used to optimize drawing. Note: when using
-       blitting any animated artists will be drawn according to their zorder.
-       However, they will be drawn on top of any previous artists, regardless
-       of their zorder.  Defaults to *False*.
+    blit : bool, default: False
+        Whether blitting is used to optimize drawing.  Note: when using
+        blitting, any animated artists will be drawn according to their zorder;
+        however, they will be drawn on top of any previous artists, regardless
+        of their zorder.
 
-    cache_frame_data : bool, optional
-       Controls whether frame data is cached. Defaults to *True*.
-       Disabling cache might be helpful when frames contain large objects.
+    cache_frame_data : bool, default: True
+        Controls whether frame data is cached.
```

Comment:
nitpick as well. You removed `Controls` for the blit argument. It would make sense to remove it here as well.